packages/foundation-models/src/azure-openai/azure-openai-embedding-response.test.ts

```diff
@@ -16,7 +15,6 @@ describe('Azure OpenAI embedding response', () => {
   const response = new AzureOpenAiEmbeddingResponse(rawResponse);
 
   it('should return the embedding response', () => {
-    const data = azureOpenAiEmbeddingOutputSchema.parse(response.data);
-    expect(data).toStrictEqual(mockResponse);
+    expect(response.data).toStrictEqual(mockResponse);
```

Comment:
I realize this test is quite useless, we should be testing the utility functions here as well. I'll fix this in another PR.

Reply:
I agree. This test just check if `data` is set. Utility functions are more important.